pydruid/utils/filters.py

```diff
@@ -67,3 +72,10 @@ def __init__(self, dim):
 
     def __eq__(self, other):
         return Filter(dimension=self.dimension, value=other)
+
+class Javascript:
```

Comment:
minor nit, in the main Druid repo we typically use `JavaScript` instead of `Javascript` in class names